smart_open/tests/test_s3.py

```diff
@@ -65,16 +66,23 @@ def create_bucket_and_key(
     #
     for attempt in range(num_attempts):
         try:
-            mybucket = s3.Bucket(bucket_name)
-            mykey = s3.Object(bucket_name, key_name)
-            if contents is not None:
-                mykey.put(Body=contents)
-            return mybucket, mykey
+            s3.Object(BUCKET_NAME, KEY_NAME).put(Body=contents)
+            return
         except botocore.exceptions.ClientError as err:
             logger.error('caught %r, retrying', err)
             time.sleep(sleep_time)
 
-    assert False, 'failed to create bucket after %d attempts' % num_attempts
+    assert False, 'failed to create bucket %s after %d attempts' % (BUCKET_NAME, num_attempts)
+
+
+def clean_and_populate_bucket(num_keys=10):
```

Comment:
Can you please move this function definition down closer to where the populate_bucket function was (L593 in previous version)? It'd make it a lot easier to review.